template/src/Placeholder.Core/Weather/IWeatherRepository.cs

```diff
@@ -4,5 +4,9 @@ namespace Placeholder.Core;
 
 public interface IWeatherRepository
 {
-    Task<Weather[]> GetWeather();
+    Task<Weather[]> GetWeathers();
```

Comment:
C'est généralement une bonne pratique d'utiliser des [interfaces vs des implémentations](https://stackoverflow.com/questions/2697783/what-does-program-to-interfaces-not-implementations-mean):

```suggestion
Task<IEnumerable<Weather> GetWeathers();
```

Généralement pour les listes, on [favorise IEnumerable](https://stackoverflow.com/questions/3433284/ienumerablet-vs-t).

Reply:
done